apps/examples/wc-next10/next.config.js

```diff
@@ -1,6 +1,9 @@
-/** @type {import('next').NextConfig} */
-const nextConfig = {
-  reactStrictMode: true,
-}
+const transpileModules = [
+    '@justeattakeaway/pie-button',
+    '@lit-labs/react',
+    'lit'
+];
 
-module.exports = nextConfig
+const withTM = require('next-transpile-modules')(transpileModules);
+
+module.exports = withTM;
```

Comment:
If you wanted to include some Next config alongside this transpilation, what would you need to do?

Reply:
It looks like you could do this potentially (from a v quick look):

`withTM({ … }`

Would be worth a check with a simple Next 10 option though and make sure you can pass through config in that way (and if so, just adding that to the README)

Reply:
You can use lodash/flow to return transpilation along with the next config -
 ```
return flow([withTM])(
      nextConfig,
    );
```